creator-node/src/components/tracks/trackContentUploadManager.ts

```diff
@@ -1,29 +1,36 @@
-const path = require('path')
+import type { LogContext } from '../../apiHelpers'
 
-const {
+import path from 'path'
+import {
   getStartTime,
   logInfoWithDuration,
-  logger: genericLogger
-} = require('../../logging')
-const { getSegmentsDuration } = require('../../segmentDuration')
-
-const models = require('../../models')
-const { libs } = require('@audius/sdk')
+  logger as genericLogger
+} from '../../logging'
+import { getSegmentsDuration } from '../../segmentDuration'
+
+import DBManager from '../../dbManager'
+import TranscodingQueue from '../../TranscodingQueue'
+import FileManager from '../../fileManager'
+import { libs } from '@audius/sdk'
```

Comment:
can we move this import next to the `path` import? let's keep the 3rd party libraries next to each other